src/block/Door.php

```diff
@@ -176,4 +177,15 @@ public function getAffectedBlocks() : array{
 	private function canBeSupportedAt(Block $block) : bool{
 		return $block->getAdjacentSupportType(Facing::DOWN)->hasEdgeSupport();
 	}
+
+	public function onWindChargeInteraction(WindCharge $windCharge) : void{
+		if($this->getTypeId() == BlockTypeIds::IRON_DOOR) {
```

Comment:
if(!self instanceof WoodenDoor){
    return;
}

Would make a bit more sense here

Reply:
careless porting from a separate file, thank you for this catch!

Reply:
but that wouldnt work beacuse of copper doors!